From 541f3b85cac00e5b836e018ef8afea4342e0cd39 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Ondruch?= <vondruch@redhat.com>
Date: Tue, 9 Jul 2013 12:32:49 +0200
Subject: [PATCH] Make stable Gem::Specification.files in default .gemspecs

Although the .gemspec files for default gems are same in function, the
different order of their "s.files" makes them different therefore
possibly conflicting in multilib scenario.
---
 tool/rbinstall.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tool/rbinstall.rb b/tool/rbinstall.rb
index 0b99408..7b54966 100755
--- a/tool/rbinstall.rb
+++ b/tool/rbinstall.rb
@@ -608,7 +608,7 @@ module RbInstall
       end
 
       def collect
-        ruby_libraries + built_libraries
+        (ruby_libraries + built_libraries).sort
       end
 
       private
-- 
1.8.3.1

